templates/theme-footer.html

```diff
@@ -20,7 +20,7 @@
             <li><a href="${reverse('support')}">Support</a></li>
             <li><a href="${reverse('privacy')}">Privacy Policy</a></li>
           </ol>
-          <div class="copyright">&copy; 2015 Aquent Gymnasium</div>
+          <div class="copyright">&copy; 2016 Aquent Gymnasium</div>
```

Comment:
@mbifulco we've started using this snippet in our footer templates:

```
<%! from datetime import datetime %>
<% current_year = datetime.now().year %>

<span>&copy; ${current_year} SomeCompany </span>
```

It'll prevent you from having to update this file once a year.

If you want to make a similar chance, we can include it in this PR if you'd like, or wait until the next one. It's up to you.